### PR TITLE
done login ui

### PR DESCRIPTION
**Pull request by buitannguyen2612**

### CODE REVIEW

File: app/components/login/page.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+blockquote,q{quotes:none}
+blockquote:before,blockquote:after,q:before,q:after{content:none}
+a{margin:0;padding:0;font-size:100%;vertical-align:baseline;background:transparent}
+ins{background-color:#ff9;color:#000;text-decoration:none}
+mark{background-color:#ff9;color:#000;font-style:italic;font-weight:bold}
+del{text-decoration:line-through}
+abbr[title],dfn[title]{border-bottom:1px dotted;cursor:help}
+table{border-collapse:collapse;border-spacing:0}
+hr{display:block;height:1px;border:0;border-top:1px solid #ccc;margin:1em 0;padding:0}
+input,select{vertical-align:middle}
```
Comment (thdailong) [inline app/components/login/page.css]: Do we really need these lines?

File: app/components/login/page.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+.container_login{
```
Comment (thdailong) [inline app/components/login/page.css]: name of class css should not be in this format
```suggestion
.container-login {
```

File: app/components/login/page.css [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    height: 100vh;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.login_box{
```
Comment (thdailong) [inline app/components/login/page.css]: Same here

File: app/components/login/page.css [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    width: 100%;
+    display: block;
+    margin-top: 20px;
+}
+
+.login__span .input{
+    width: 100%;
+}
+
+.login_btn{
```
Comment (thdailong) [inline app/components/login/page.css]: Same here

File: app/components/login/page.css [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    background-color: white;
+}
+
+.login__span{
+    width: 100%;
+    display: block;
+    margin-top: 20px;
+}
+
+.login__span .input{
```
Comment (thdailong) [inline app/components/login/page.css]: Same here

File: app/components/login/page.css [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+    display: flex;
+    flex-direction: column;
+    box-shadow: #c5c5c5 0px 2px 8px 3px;
+    border-radius: 10px;
+    gap: 10px;
+    padding: 18px 18px;
+    background-color: white;
+}
+
+.login__span{
```
Comment (thdailong) [inline app/components/login/page.css]: Same here

File: app/components/login/page.jsx [hunk 7]
```diff
@@ -0,0 +1,93 @@
+"use client"; 
+import React, { useState } from "react";
+import { Button, TextField } from "@mui/material";
+import "./page.css";
```
Comment (thdailong) [inline app/components/login/page.jsx]: you can search for css module!
https://nextjs.org/docs/app/building-your-application/styling/css-modules

File: app/components/login/page.jsx [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+  }
+
+
+  //ham handle click login button
+  const submitLogin = (e) => {
+    e.preventDefault()
+  }
+
+  return (
+    <div className="container_login">
```
Comment (thdailong) [inline app/components/login/page.jsx]: As we already have "Box" component in material UI
You should use "Box" instead of "div"

**Review by thdailong — CHANGES_REQUESTED** [review]
Please fix all!

File: app/components/login/page.css (hunk 1 above)
Comment (buitannguyen2612) [reply app/components/login/page.css]: so this is reset css, still need it ?

File: app/components/login/page.css (hunk 2 above)
Comment (buitannguyen2612) [reply app/components/login/page.css]: confirm

File: app/components/login/page.jsx (hunk 7 above)
Comment (buitannguyen2612) [reply app/components/login/page.jsx]: confirm

File: app/components/login/page.jsx (hunk 8 above)
Comment (buitannguyen2612) [reply app/components/login/page.jsx]: confirm

File: app/components/login/page.css (hunk 1 above)
Comment (phunlh2001) [reply app/components/login/page.css]: I think CSS BaseLine in MUI will reset CSS instead

File: app/auth/FogotPassword/page.jsx [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+import Link from "next/link";
+
+export default () => {
+
+    const [emails, setEmail] = useState('')
+    const [emilValid, setEmailValid] = useState(true)
+
+
+    // regex check valid email
+    const validateEmail = (email) => {
```
Comment (thdailong) [inline app/auth/FogotPassword/page.jsx]: You could move this into a new folder utils/utils.js.
As we could this function many times

File: app/auth/FogotPassword/page.module.css [hunk 10]
```diff
@@ -0,0 +1,11 @@
+.imgicon{
```
Comment (thdailong) [inline app/auth/FogotPassword/page.module.css]:
```suggestion
.img-icon{
```

File: app/auth/FogotPassword/page.module.css [hunk 11]
```diff
@@ -0,0 +1,11 @@
...
+    height: 100%;
+    width: 100%;
+    border-radius: 100%;
+}
+
+.icontext{
```
Comment (thdailong) [inline app/auth/FogotPassword/page.module.css]:
```suggestion
.icon-text{
```

File: app/auth/FogotPassword/page.jsx [hunk 12]
```diff
@@ -0,0 +1,90 @@
+"use client";
+import { Box, Button, TextField } from "@mui/material";
+import Layout from "../layput";
+import imgageicon from '../images/forgotIcon.png'
```
Comment (thdailong) [inline app/auth/FogotPassword/page.jsx]: I think we should save the image as svg.
It will make the website render image faster

File: app/auth/layput.js
Comment (thdailong) [inline app/auth/layput.js:1]: layput.js???

**Review by thdailong — CHANGES_REQUESTED** [review]
Please fix!

File: app/auth/FogotPassword/page.jsx (hunk 12 above)
Comment (buitannguyen2612) [reply app/auth/FogotPassword/page.jsx]: ok, confirm that

**Review by thdailong — APPROVED** [review]

**Pull request merged**